tests/notification/test_validation.py

```diff
@@ -0,0 +1,17 @@
+from django.test import TestCase
+from django.core.exceptions import ValidationError
+
+from callisto.notification.models import EmailNotification
+
+
+class EmailValidationTest(TestCase):
+
+    def test_duplicate_emails_not_allowed(self):
```

Comment:
glad you poked me to add more tests, because I the test I wrote wasn't testing the logic well enough